### PR TITLE
Schedule hangout default studygroup name

### PR DESCRIPTION
**Pull request by railsstudent**
When I schedule new hangout in single study group, study group name is not default in the new hangout modal. I have to type study group name in the drop down to narrow down the selections and select the group name manually.

Similarly, it also occurs when duplicating hangout in hangout page and single study group. This change will populate study group name in addition to description and topic

### CODE REVIEW

File: client/templates/hangout/hangout-clone.js [hunk 1]
```diff
@@ -53,6 +53,10 @@ Template.cloneHangoutModal.onRendered(function() {
         data: studyGroups
       });
 
+      const groupId = templateInstance.data.hangout.data.group.id || '';
+      if (groupId !== '') {
+    -   instance.$(".study-group-single").val(groupId).trigger('change');
```
Comment (lpatmo) [inline client/templates/hangout/hangout-clone.js]: This is awesome, @railsstudent. Thanks for addressing this issue.

Question: Was this `-` on line 58 added by mistake, or am I missing something?

Comment (railsstudent) [reply client/templates/hangout/hangout-clone.js]: It is a typo. Added by mistake.

Comment (railsstudent) [reply client/templates/hangout/hangout-clone.js]: I will make the correction tonight

Comment (railsstudent) [reply client/templates/hangout/hangout-clone.js]: @lpatmo  Fixed. Please kindly review

**Pull request merged**